apache-demo/docs/adopt.md

```diff
@@ -0,0 +1,177 @@
+This example shows you how to use this resource package, IBM® Telco Network Cloud - Orchestration and the OpenStack VIM driver to adopt a stack in OpenStack. 
+
+### Before you begin
+The variables that are used in this procedure are described in the following table:
+
+*Table 1. Variables used in this procedure*
+
+| Variable	| Description |
+|---|---|
+| $NAMESPACE | The namespace in which IBM Telco Network Cloud - Orchestration is installed. For example, lifecycle-manager. |
+| $PROXY_ADDRESS | Public API address, for example, alm-ishtar-lifecycle-manager.ocp.ibm.com.| 
+| $CLIENT_ID	API | username. Admin by default.| 
+| $CLIENT_SECRET	| The key for $CLIENT_ID.| 
```

Comment:
`API | username` -> `| API username`?

Reply:
I suggest in my review possibly removing the list of variables at the beginning altogether and referencing the Invoking REST API KC docs - similar to the apache-ansible example.